Core/Src/segment.c

```diff
@@ -5,8 +5,55 @@
 #include <stdlib.h>
 #include <string.h>
 
+// TEMPORARY
+
+#include "common.h"
+#include "adBms6830CmdList.h"
+#include "adBms6830GenericType.h"
+#include "serialPrintResult.h"
+#include "adBms6830ParseCreate.h"
+#include "mcuWrapper.h"
+
+#define ALL_GPIOS_ARE_INPUTS 0x3FF
+
+typedef enum {
+	DISCHARGE_ENABLED = 0,
+	MUTE_ACTIVATED_DISCHARGE_DISABLED = 1
+} MUTE_ST;
+
+#define TOTAL_IC 1
+cell_asic IC[TOTAL_IC];
+
+RD REDUNDANT_MEASUREMENT = RD_OFF;
```

Comment:
we do want redundant in final rev

Reply:
Sure we can do that later. I want a full spec from BMS team on exactly what the configs should be and how they want the BMS to work